packages/cli/src/service/pipeline.ts

```diff
@@ -72,10 +72,8 @@ export async function installPackageFromConfig(config: any, opts: any): Promise<
   for (const plugin of constants.PLUGINS) {
     const packageName = config.plugins[plugin]?.package;
     if (typeof packageName === 'string') {
-      if (packageName[0] === '.') {
-        const pkg = await pluginInstall(packageName, opts);
-        config.plugins[plugin].package = pkg.name;
-      }
+      const pkg = await pluginInstall(packageName, opts);
```

Comment:
This might be better to put in another PR :)

Reply:
This fixes an error that would not have been installed if the  name was not a local path.

Reply:
What error on that case?

Reply:
`installPackageFromConfig ` should install plugins whatever the plugin name protocol(local path, url, git or npm package name).
But it accepts local path only.

Reply:
Ok, resolved.